data/gigadb_testdata/author.csv

```diff
@@ -25,3 +25,14 @@ id,surname,middle_name,first_name,orcid,gigadb_user_id,custom_name
 3807,Vattikuti,"",S,"",,
 3808,Alnasir,"",J,"",,
 3809,Shanahan,"",H,"",,
+14014,Ou,"",Mi,"",,
+14015,Huang,"",Rong,"",,
+14019,Yang,"",Cheng,"",,
+14020,Gui,"",Bin,"",,
+14016,Luo,"",Qing,"",,
+14017,Zhao,"",Jian,"",,
+14021,Li,"",Yongming,"",,
+14022,Liao,"",Lanjie,"",,
+14023,Zhu,"",Zuoyan,"",,
+14013,Wang,"",Yaping,"",,
+14018,Chen,"",Kunci,"",,
```

Comment:
I think it would be good to have the new author rows listed in order of `id` like the other author rows already there.